codersdk/roles.go

```diff
@@ -9,16 +9,48 @@ import (
 	"github.com/google/uuid"
 )
 
-type Role struct {
+type SlimRole struct {
 	Name        string `json:"name"`
 	DisplayName string `json:"display_name"`
 }
 
 type AssignableRoles struct {
-	Role
+	SlimRole
 	Assignable bool `json:"assignable"`
 }
```

Comment:
Can you provide a comment explaining the difference between `Role` and `SlimRole`?